src/style.css

```diff
@@ -0,0 +1,130 @@
+* {
```

Comment:
- [ ] You have an empty print.js file. Please remove it if you intend not to use it